obtain_new_ids.py

```diff
@@ -301,11 +301,10 @@ class BKCIIDFinder(IDFinder):
 
     def get_query_parameters(self):
         """Construct Thoth work ID query parameters depending on Clarivate BKCI-specific requirements"""
-        # Target: all active (published) works listed in Thoth (from the selected publishers).
+        # Target: all active (published) works listed in Thoth (from the selected publishers), except for textbooks
         self.work_statuses = '[ACTIVE]'
-        # Textbooks not accepted
         self.work_types = '[MONOGRAPH, EDITED_BOOK, JOURNAL_ISSUE, BOOK_SET]'
-        # Start with the earliest, so that the upload is logically ordered
+        # Start with the most recent, so that the upload is logically ordered
```

Comment:
Compare the comment at line 88 - the ordering is necessary for the subsequent code to work correctly